doc/languages-frameworks/python.md

```diff
@@ -247,23 +255,27 @@ Python bindings to `libxml2` and `libxslt`. These libraries are only required
 when building the bindings and are therefore added as `buildInputs`.
 
 ```nix
-lxml = buildPythonPackage rec {
-  name = "lxml-3.4.4";
-
-  src = pkgs.fetchurl {
-    url = "mirror://pypi/l/lxml/${name}.tar.gz";
-    sha256 = "16a0fa97hym9ysdk3rmqz32xdjqmy4w34ld3rm3jf5viqjx65lxk";
-  };
+{
+  # ...
+  lxml = buildPythonPackage rec {
+    name = "lxml-3.4.4";
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/l/lxml/${name}.tar.gz";
+      sha256 = "16a0fa97hym9ysdk3rmqz32xdjqmy4w34ld3rm3jf5viqjx65lxk";
+    };
 
-  buildInputs = with self; [ pkgs.libxml2 pkgs.libxslt ];
+    buildInputs = with self; [ pkgs.libxml2 pkgs.libxslt ];
```

Comment:
No, I mean we should ask the python guys from nixpkgs to ensure that we can remove `self` here.

`self` is commonly used to refer to the attribute set currently in, so this would bring everything from the current set into the scope of the expression. But my guess is that we do not need it, as we do not refer to something from `self`, like `self.somepackage`, but rather to `pkgs.somepackage` - so bringing `pkgs` into scope like so:

```nix
with pkgs; [ libxml2 libxslt ];
```

would make more sense to me. But as said: I'm not into the python-packaging ecosystem that much, so I would say we need someone from the python packaging team to approve what I suggest.